fluid_onnx/ops.py

```diff
@@ -687,6 +682,7 @@ def xor_op():
     'conv2d': conv2d_op,
     # Need to continue the mapping below.
     'conv2d_transpose': conv2d_transpose_op,
+    # 'cos': partial(activation_ops, 'Cos'),
```

Comment:
Sorry for missing this one!

Reply:
@varunarora Thanks! It is OK, I just added these ops in passing :-)